static/app/components/searchQueryBuilder/index.spec.tsx

```diff
@@ -396,6 +396,36 @@ describe('SearchQueryBuilder', function () {
         expect(recentFilterKeys[1]).toHaveTextContent('is');
       });
 
+      it('does not display recent filters that are not valid filter keys', async function () {
+        MockApiClient.addMockResponse({
+          url: '/organizations/org-slug/recent-searches/',
+          body: [
+            {query: 'assigned:me'},
+            {query: 'assigned:me browser.name:firefox'},
+            {query: 'assigned:me browser.name:firefox is:unresolved'},
+            // Level is not a valid filter key
+            {query: 'assigned:me browser.name:firefox is:unresolved level:error'},
```

Comment:
For this test, can you remove the first few queries and simplify this one to `assigned:me level:error`? We only return the top 3 filters now, so while the test passes I don't think it would fail if you took out the new logic new logic.

Reply:
Got it, will make the change.